### ISSUE TITLE
Improve the json files management

### ISSUE THREAD

**Issue by maxwab** [axondeepseg/axondeepseg#27]
Right now we use one configuration file (config_network.json) where every information about the training hyperparameters, the network architecture and the dataset information is stored.

Yet the dataset information (like the size of the patches that compose the dataset) is a static information that is shared between models. In addition, when we apply a model, we only need to know its architecture, and not the value of the learning rate.

_Suggestion:_
* Split config_network.json in 3 different configurations files : parameters, architecture and dataset.
* Adapt the code to be able to manage this new organization (not that trivial).

This is not urgent.

**Comment by jcohenadad** [comment]
in relation to https://github.com/neuropoly/axondeepseg/issues/73

**Comment by mariehbourget** [comment]
I think we can close this issue as the `config_network.json` file does not exist anymore with the ivadomed implementation.
